test/uintvar.cpp

```diff
@@ -11,16 +12,36 @@ namespace var {
 
 TEST_CASE("Encode/Decode UintVar")
 {
-    CHECK_EQ(var::kValue1Byte, quicr::ToUint64(quicr::ToUintV(var::kValue1Byte)));
-    CHECK_EQ(var::kValue2Byte, quicr::ToUint64(quicr::ToUintV(var::kValue2Byte)));
-    CHECK_EQ(var::kValue4Byte, quicr::ToUint64(quicr::ToUintV(var::kValue4Byte)));
-    CHECK_EQ(var::kValue8Byte, quicr::ToUint64(quicr::ToUintV(var::kValue8Byte)));
+    CHECK_EQ(var::kValue1Byte, uint64_t(quicr::UintVar(var::kValue1Byte)));
+    CHECK_EQ(var::kValue2Byte, uint64_t(quicr::UintVar(var::kValue2Byte)));
+    CHECK_EQ(var::kValue4Byte, uint64_t(quicr::UintVar(var::kValue4Byte)));
+    CHECK_EQ(var::kValue8Byte, uint64_t(quicr::UintVar(var::kValue8Byte)));
 }
 
 TEST_CASE("Length of UintVar")
 {
-    CHECK_EQ(1, quicr::SizeofUintV(quicr::ToUintV(var::kValue1Byte).at(0)));
-    CHECK_EQ(2, quicr::SizeofUintV(quicr::ToUintV(var::kValue2Byte).at(0)));
-    CHECK_EQ(4, quicr::SizeofUintV(quicr::ToUintV(var::kValue4Byte).at(0)));
-    CHECK_EQ(8, quicr::SizeofUintV(quicr::ToUintV(var::kValue8Byte).at(0)));
-}
\ No newline at end of file
+    CHECK_EQ(1, quicr::UintVar(var::kValue1Byte).Size());
+    CHECK_EQ(2, quicr::UintVar(var::kValue2Byte).Size());
+    CHECK_EQ(4, quicr::UintVar(var::kValue4Byte).Size());
+    CHECK_EQ(8, quicr::UintVar(var::kValue8Byte).Size());
+
+    CHECK_EQ(1, quicr::UintVar(var::kValue1Byte).Bytes().size());
```

Comment:
The previous test was testing getting the size with a single byte (e.,g reading in partial data), this test requires the value to have been set, which required `UintVar(all bytes)` first.   The use-case is still there for checking the size of the uintvar by reading the first byte, which drives how many bytes are needed to load the uintvar.   This test is fine, but a new test is needed to replace what the old one was doing.

Reply:
That is what the test on line 26 is handling, it just does it under the hood for you instead. This si separate test to make sure that the Bytes span size is also correct.